.aws/first.sh

```diff
@@ -15,7 +15,7 @@ NODEID=$(./gaiad tendermint show-node-id --home .)
 
 # Create our main account and add it to the genesis with a lot of money
 echo ${PASSWORD} | ./gaiacli keys add ${ACCOUNT} --home . > account_address.log
-./gaiad add-genesis-account $(./gaiacli keys show ${ACCOUNT} --home . --address) 100000000000000000stake,999000000000photino,123123123123cococoin --home .
+./gaiad add-genesis-account $(./gaiacli keys show ${ACCOUNT} --home . --address) 1000000000000000000000stake,100000000000000000000photino,123123123123123123cococoin --home .
```

Comment:
why not 9999999999999999999999999999999 so we don't run out

Reply:
18446744073709551615 should be the biggest available

[](https://tour.golang.org/basics/11)

Reply:
lol ok why not that number then?

Reply:
we are already overflowing XD:
```
1000000000000000000000
18446744073709551615
```
So probably we already have everything we can

Reply:
My bad, on SDK I found out big integers are used so we have way more than 64 bits, as soon as I'm sure about the number of bits I'll update this number